app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.html

```diff
@@ -3,7 +3,7 @@
 
   <ng-template #tooltipPopover>
     <span>
-      <strong> {{ getStepText() }} </strong>
+      <strong [innerHtml]="getStepText()"></strong>
```

Comment:
is there any way we can assign this value at `OnInit()` or at leats make it observable somehow?  The issue here is that on each pass of the change detector loop we're executing [this script block](https://github.com/gashcrumb/syndesis/blob/99cc1a514695b621062a65dee6df357acba88706/app/ui/src/app/integration/edit-page/flow-view/flow-view-step.component.ts#L273-L298), which seems a bit demanding to me.

Reply:
Yep, was planning on tackling that kind of thing while implementing the changes for data buckets, this one is a first-pass untangling :-)